uisdk/src/main/res/layout/uisdk_view_booking_terms.xml

```diff
@@ -33,23 +34,26 @@
     <TextView
         android:id="@+id/termsConditionsText"
         style="@style/KhBookingTermsView"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toStartOf="@id/guidelineCentre"
-        app:layout_constraintTop_toBottomOf="@id/bookingTermsText"
+        android:layout_width="163dp"
+        android:layout_height="0dp"
```

Comment:
I think we need to make sure the height is at lest 48dp to meet accessibility guidelines. Also, I don't think we should set the width in dp here as it'll need to vary based on screen size. Same goes for below.